@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fn_getInput.sqf

```diff
@@ -45,6 +45,11 @@ private _pedalLeftRight    = (inputAction "HeliRudderRight") - (inputAction "Hel
 private _tailRtrDamage     = _heli getHitPointDamage "hitvrotor";
 
 private _collectiveOut     = 0.0;
+private _prevCollectiveOut = _heli getVariable "fza_sfmplus_prevCollectiveOut";
+
+private _pauseMenu		   = findDisplay 49;
+if (!isNull _pauseMenu) exitwith { fza_sfmplus_collectiveOutput = 0.29; fza_inMpPauseMenu = true; };//fza_sfmplus_collectiveOutput should hold whatever the previous collectiveOut value was...
```

Comment:
Why is there a hard-coded collective position?

Could you use indentation and not make that a one-liner as with the condition, operations, and comment that line is very long. I also try to put each operation on a new line so they can't be missed